src/test/test_addr.c

```diff
@@ -763,6 +763,8 @@ test_addr_ip6_helpers(void *arg)
   tt_int_op(r, OP_LE, 0); // "it worked or it didn't"
   i = get_interface_address6(LOG_DEBUG, AF_INET6, &t2);
   tt_int_op(i, OP_LE, 0); // "it worked or it didn't"
+  r = get_interface_address6(LOG_DEBUG, AF_INET, &t1);
+  i = get_interface_address6(LOG_DEBUG, AF_INET6, &t2);
```

Comment:
Why add these extra unit tests?
They don't seem to do anything different to the lines above them.